lib/common/models/wp_version/output.rb

```diff
@@ -4,8 +4,12 @@ class WpVersion < WpItem
   module Output
 
     def output(verbose = false)
+      metadata = self.metadata(self.number)
+
       puts
       puts info("WordPress version #{self.number} identified from #{self.found_from}")
+      puts " | Released: #{metadata[:release_date]}"
+      puts " | Changelog: #{metadata[:changelog_url]}"
```

Comment:
do we really need to show the changelog of the installed version? The interesting part would be the changelog of follow up versions, but we can't show this. So I would just put the release date in the Wordpress identified string.